contracts/VotingEscrow.vy

```diff
@@ -139,6 +143,16 @@ def __init__(token_addr: address, _name: String[64], _symbol: String[32], _versi
     self.version = _version
 
 
+@external
+def set_is_active(val: bool):
```

Comment:
how about `set_breaker(break: bool):`
so that default is false and doesn't require initialization which is a common approach